feat: rename template to snapshot and add initial files for classic LVM snapshot support (#2)
linux-system-roles/snapshot: README.md

![template](https://github.com/linux-system-roles/snapshot/workflows/tox/badge.svg)

The `snapshot` role enables users to add/remove snapshots on target machines.
This role can be used to configure snapshots via:

- LVM

## Requirements

> Managed node must have python and LVM installed

Reviewer: The role should install this, so we don't need to mention it here.

Author: Ok

> # linux-system-roles/snapshot

Reviewer 2: ```suggestion
# snapshot
```

![template](https://github.com/linux-system-roles/snapshot/workflows/tox/badge.svg)

The `snapshot` role enables users to add/remove snapshots on target machines.
This role can be used to configure snapshots via:

- LVM

> ## Requirements

Reviewer 2: ```suggestion
```

### Collection requirements

## Role Variables

### snapshot_lvm_action

> This variable is required.  The supported values are snapshot, check, clean.

Reviewer 2: ```suggestion
This variable is required. It supports one of the following values:
- `snapshot`: Take snapshots of the specified VGs/LVs
- `check`: Validate that snapshot names don't have conflicts and there is sufficent space to take the snapshots
- `clean`: Remove snapshots that conform to the specified prefix and pattern
```

Changed to use a clearer structure and present simple tense for simplicity

  lv2_vg2 vg2  -wi-a-----  80.00m
  lv1_vg3 vg3  -wi-a-----   1.00g
  lv3_vg3 vg3  -wi-a----- 120.00m

If the prefix is set to "a_" and the suffix is set to "_z", running the role will result
in the following:

  LV          VG   Attr       LSize   Pool Origin  Data%  Meta%  Move Log Cpy%Sync Convert
  a_home_z    rhel swi-a-s--- 104.00m      home    0.00
  a_root_z    rhel swi-a-s---   3.50g      root    0.01
  a_swap_z    rhel swi-a-s--- 400.00m      swap    0.00
  home        rhel owi-aos---   1.00g
  root        rhel owi-aos---  35.00g
  swap        rhel owi-aos---  <3.88g
  a_lv1_vg1_z vg1  swi-a-s--- 104.00m      lv1_vg1 0.00
  a_lv2_vg1_z vg1  swi-a-s---   8.00m      lv2_vg1 0.00
  lv1_vg1     vg1  owi-a-s---   1.00g
  lv2_vg1     vg1  owi-a-s---  40.00m
  a_lv1_vg2_z vg2  swi-a-s--- 104.00m      lv1_vg2 0.00
  a_lv2_vg2_z vg2  swi-a-s---  12.00m      lv2_vg2 0.00
  lv1_vg2     vg2  owi-a-s---   1.00g
  lv2_vg2     vg2  owi-a-s---  80.00m
  a_lv1_vg3_z vg3  swi-a-s--- 104.00m      lv1_vg3 0.00
  a_lv3_vg3_z vg3  swi-a-s---  16.00m      lv3_vg3 0.00
  lv1_vg3     vg3  owi-a-s---   1.00g
  lv3_vg3     vg3  owi-a-s--- 120.00m

### snapshot_lvm_percent_space_required

> This is required for check and snapshot commands.

Reviewer 2: ```suggestion
This is required for check and snapshot actions.
```
